third_party/tlx/dialect/triton_tlx.cc

```diff
@@ -274,10 +275,18 @@ void init_triton_tlx_ir(py::module &&m) {
                }
                // TODO: find the defining op properly - the defining op is not
                // necessarily MemDescSubviewOp
-               auto subviewOp = value.getDefiningOp<ttg::MemDescSubviewOp>();
-               assert(subviewOp && "Failed to find TMEMAllocOp defining the "
-                                   "accumulator TMEM passed to dot op.");
-               value = subviewOp.getSrc();
+               auto definingOp = value.getDefiningOp();
```

Comment:
This searching logic could be replaced with `getBackwardSlice` if the logic ever went more complicated.